.github/workflows/release.yaml

```diff
@@ -0,0 +1,63 @@
+on:
+  workflow_dispatch:
+    branches: main
+
+jobs:
+  release:
+    name: Perform release
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git config --global user.email github-actions@github.com
+          git config --global user.name github-actions
+      - name: Cocogitto release
+        id: release
+        uses: oknozor/cocogitto-action@v3.4
+        with:
+          release: true
+          git-user: 'github-actions'
+          git-user-email: 'github-actions@github.com'
+      - name: Generate Changelog
```

Comment:
What is the difference with [GitHub release note generator](https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes)?